honeybee_grasshopper_energy/src/HB Search Materials.py

```diff
@@ -0,0 +1,57 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Search for available Materials within the honeybee energy standards library.
```

Comment:
Very much like the idea of separating the search components.

Reply:
Glad you think so!